app/auth/forms.py

```diff
@@ -4,14 +4,33 @@
 from flask_babel import _, lazy_gettext as _l
 from app.models import User
 
-
 class LoginForm(FlaskForm):
     username = StringField(_l('Username'), validators=[DataRequired()])
     password = PasswordField(_l('Password'), validators=[DataRequired()])
     remember_me = BooleanField(_l('Remember Me'))
     submit = SubmitField(_l('Sign In'))
 
 
+class Confirm2faForm(FlaskForm):
+    token = StringField('Token')
+    submit = SubmitField('Verify')
+
+
+class Enable2faForm(FlaskForm):
+    verification_phone = StringField('Phone', validators=[DataRequired()])
+    submit = SubmitField('Enable 2FA')
+
+    def validate_verification_phone(self, verification_phone):
+        try:
+            return
+        except:
+            print("An exception occurred")
```

Comment:
1. if you capture an error, you should print out what it should be.
2. this function literally does nothing so well it shouldnt exist.

Reply:
It used to have functionality but due to being unable to import the phonenumbers package, I removed some lines for testing and forgot to remove the try except entirely. I'll remove it.